part1/restaurant.rb

```diff
@@ -0,0 +1,12 @@
+class Restaurant
+  def initialize(menu)
+    @menu = menu
+  end
+
+  def cost(*orders)
+    cost = orders.map do |order|
+      order[:rice] * @menu[:rice] + order[:noodles] * @menu[:noodles]
```

Comment:
it looks like you broke Law of Demeter with this, as you use only `menu[:rice]`  `@menu[:noodles]` but have object menu, which kind of excessive

pls look read about  Law of Demeterhttp://www.virtuouscode.com/2011/07/05/demeter-its-not-just-a-good-idea-its-the-law/